app/init/init_gatekeeper.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from core import settings
+
+def register_apis_to_gatekeeper():
+
+    # Login
+    at = requests.post(
+        url=settings.GATEKEEPER_BASE_URL.unicode_string() + "api/login/",
+        headers={"Content-Type": "application/json"},
+        json={"username": "admin", "password": "admin"}
+    )
+
+    temp = at.json()
+
+    access = temp["access"]
+    refresh = temp["refresh"]
+
+    # Register APIs
+    # APIs shouldn't have slashes on their ends even though they do have them
+    apis_to_register = [
```

Comment:
ideally this should also be created by iterating the `FastAPI.routes` object

Reply:
Nice idea! Didn't cross my mind previously, that would be much better actually, since one can add more and remove other apis.
I need to see how it prints them out and how to use these in the registration.